src/rules/noAnyRule.ts

```diff
@@ -33,7 +33,7 @@ export class Rule extends Lint.Rules.AbstractRule {
     };
     /* tslint:enable:object-literal-sort-keys */
 
-    public static FAILURE_STRING = "Type declaration of 'any' is forbidden";
+    public static FAILURE_STRING = "Type declaration of 'any' is forbidden (use the empty type '{}'?)";
```

Comment:
> Separately, it would be nice if the end of a tslint failure report pointed you to the docs for suppression https://palantir.github.io/tslint/usage/rule-flags/ - nearly every user I've coached was unaware of those.

I don't think bloating the failure message even more adds any more value to it. Most of the time you see it as a tooltip in your editor and cannot even click the link.

Some (most?) IDEs with tslint integration show you a "quickfix" option at rule failures that allows you to automatically disable that rule for that line. At least that's what vscode, IntelliJ and friends do.